apps/web-giddh/src/app/search/components/search-grid/search-grid.component.html

```diff
@@ -15,188 +15,122 @@
-                        <td [attr.data-title]="localeData?.parent">{{ item.parent }}</td>
-                        <td [attr.data-title]="localeData?.opening_balance" class="text-right">
-                            <span class="d-inline-flex">
-                                <amount-field
-                                    [amount]="item?.openingBalance"
-                                    [currencySymbol]="false"
-                                    [currencyCode]="false"
-                                >
-                                </amount-field>
-                            </span>
-                            <span *ngIf="item.openingBalance > 0">
-                                <span class="text-success" *ngIf="item.openBalanceType === 'DEBIT'">{{
-                                    localeData?.balance_type.dr
-                                }}</span>
-                                <span class="text-danger" *ngIf="item.openBalanceType === 'CREDIT'">{{
-                                    localeData?.balance_type.cr
-                                }}</span>
-                            </span>
-                        </td>
-                        <td [attr.data-title]="localeData?.dr_total" class="text-right">
-                            <span class="d-inline-flex">
-                                <amount-field
-                                    [amount]="item?.debitTotal"
-                                    [currencySymbol]="false"
-                                    [currencyCode]="false"
-                                >
-                                </amount-field>
-                            </span>
-                        </td>
-                        <td [attr.data-title]="localeData?.cr_total" class="text-right">
-                            <span class="d-inline-flex">
-                                <amount-field
-                                    [amount]="item?.creditTotal"
-                                    [currencySymbol]="false"
-                                    [currencyCode]="false"
-                                >
-                                </amount-field>
-                            </span>
-                        </td>
-                        <td [attr.data-title]="localeData?.closing_balance" class="text-right">
-                            <span class="d-inline-flex">
-                                <amount-field
-                                    [amount]="item?.closingBalance"
-                                    [currencySymbol]="false"
-                                    [currencyCode]="false"
-                                >
-                                </amount-field>
-                            </span>
-                            <span *ngIf="item.closingBalance > 0">
-                                <span class="text-success" *ngIf="item.closeBalanceType === 'DEBIT'">{{
-                                    localeData?.balance_type.dr
```

Comment:
Please add this condition in amount-field for cr and dr

Reply:
already added